### ISSUE TITLE
Speechcommands12 does not use data of "unknown" category

### ISSUE THREAD

**Issue by WWWWWLI** [The-Learning-Machines/Audiomer-PyTorch#3]
Hi
Thank you for exposing the source code.
I have a question about the use of the Google Speech Commands Dataset.
The SPEECHCOMMANDS provided in Torchaudio directly returns the label of the vocabulary without marking ["bed", "bird", "cat", "dog", "happy", "house", "Marvin", "Sheila", "tree", "wow",'visual ',' zero ',' one ',' two ',' three ',' four ',' five ',' six ',' seven ',' eight ',' nine '] as “_ unknown_”.
Lines 138 to 140 of datamodules/speechcommands12.py exclude all words that are not in the labels list (defined in line 29-40).
This results in the use of SC12 training model without any data marked “_ unknown_”.
This is inconsistent with the description of the Google Speech Commands Dataset..

**Comment by ojus1** [comment]
Thank you for pointing it out!

**Comment by ojus1** [comment]
The commit a1ce5c84ff3a5e9535d99c2fc113fa0c698b898d fixes this issue. We'll update the numbers in the preprint. Thanks!
